nautilus_trader/adapters/interactive_brokers/parsing/instruments.py

```diff
@@ -87,9 +105,10 @@ def parse_equity_contract(details: ContractDetails) -> Equity:
         price_increment=Price(details.minTick, price_precision),
         multiplier=Quantity.from_int(
             int(details.contract.multiplier or details.mdSizeMultiplier),
-        ),  # is this right?
+        ),  # TODO - is this right?
```

Comment:
Applies only to Futures and Options. We can either set to `Quantity.from_int(1)` or leave None.